### ISSUE TITLE
Cannot replicate the layer tensor values

### ISSUE THREAD

**Issue by jcgraciosa** [Jongchan/tensorflow-vdsr#25]
I have tested the code with the 'VDSR_adam_epoch_016.ckpt-57188' provided and it worked well. However, when I trained it from scratch, I noticed that the tensor values of each layer are not equal to those in the provided checkpoint. I performed the training using the 291 data set and used the parameters given in VDSR.py (e.g. LR_RATE =  0.2, and BASE_LR = 0.00001, etc.).

Do you have any insights as to why this happened?

**Comment by Jongchan** [comment]
What do you mean by "tensor values"? If it means the exact weight/activation values, it may not be identical. Because the seed is not fixed, and I heard that even when the random seed is fixed, the values may vary...

Today I noticed that the default learning rate is incorrect, and updated it.
Please run it from scratch again. 

To make sure, I am going to reproduce the numbers reported in the README.

**Comment by Jongchan** [comment]
I've run the updated code, and the results are reproducible, yet still below the numbers reported in the paper.
just use 0.0001 as default learning rate and Adam as default optimizer.
